### PR TITLE
Generate jwt token on command line

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Generate jwt token on command line
# Purpose
Currently, it takes time and effort to generate JWT tokens.
So as to easily generate jwt tokens
# Contents
Create command line tool for jwt generation and server startup using cobra
# Testing Methods Performed
CI
Create a bts image and execute the following command 
```
docker run --rm \
    -v ./path/to/directory_with_config.yml:<mount_path> \
    bts_image ./beaver_triple_service generateJwt \
    --file /path/to/config.yml \
    --output <mount_path>
```

### CODE REVIEW

File: packages/server/beaver_triple_service/cmd/generate_jwt.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	jwt_types "github.com/acompany-develop/QuickMPC/packages/server/beaver_triple_service/jwt"
+)
+
+type Options struct {
+    file_path  string
+    output_dir string
+}
+
+var (
+    o = &Options{}
```
Comment (arukuka) [inline packages/server/beaver_triple_service/cmd/generate_jwt.go:29]:
```suggestion
    options = &Options{}
```

File: packages/server/beaver_triple_service/cmd/generate_jwt.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+)
+
+func store_client_env(path string, token string) error {
```
Comment (arukuka) [inline packages/server/beaver_triple_service/cmd/generate_jwt.go:32]:
```suggestion
func storeClientEnv(path string, token string) error {
```

File: packages/server/beaver_triple_service/cmd/generate_jwt.go [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+	writer := bufio.NewWriter(fp)
+	defer writer.Flush()
+
+	fmt.Fprintf(writer, "BTS_TOKEN=%s\n", token)
+
+	return nil
+}
+
+func store_server_env(path string, secrets string, claim jwt_types.Claim) error {
```
Comment (arukuka) [inline packages/server/beaver_triple_service/cmd/generate_jwt.go:47]:
```suggestion
func storeServerEnv(path string, secrets string, claim jwt_types.Claim) error {
```

File: packages/server/beaver_triple_service/cmd/root.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+}
```
Comment (arukuka) [inline packages/server/beaver_triple_service/cmd/root.go:28]:
```suggestion
```

File: packages/server/beaver_triple_service/go.mod [hunk 5]
```diff
@@ -14,6 +14,12 @@ require (
...
 )
 
+require (
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/spf13/cobra v1.6.1 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+)
+
 require (
 	github.com/golang/protobuf v1.5.0 // indirect
```
Comment (arukuka) [inline packages/server/beaver_triple_service/go.mod:24]: Together with the require clause below.

File: packages/server/beaver_triple_service/cmd/generate_jwt.go (hunk 2 above)
Comment (arukuka) [reply packages/server/beaver_triple_service/cmd/generate_jwt.go:32]: it is written by me :bow:

**Review by arukuka — COMMENTED** [review]
Thank you!
I commented on the point of concern, so please check it.

File: packages/server/beaver_triple_service/cmd/generate_jwt.go [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	jwt_types "github.com/acompany-develop/QuickMPC/packages/server/beaver_triple_service/jwt"
+)
+
+type Options struct {
+    file_path  string
+    output_dir string
```
Comment (arukuka) [inline packages/server/beaver_triple_service/cmd/generate_jwt.go:25]: Could you use tab for indent?

File: packages/server/beaver_triple_service/cmd/generate_jwt.go (hunk 1 above)
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/cmd/generate_jwt.go]: Thanks for review!
I corrected with [f0ec7c6](https://github.com/acompany-develop/QuickMPC/pull/123/commits/f0ec7c68306dce0f30f1f77f8f355c89a4f89bb1)

I would appreciate your confirmation.

File: packages/server/beaver_triple_service/cmd/generate_jwt.go (hunk 2 above)
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/cmd/generate_jwt.go]: Corrected with [2db2237](https://github.com/acompany-develop/QuickMPC/pull/123/commits/2db2237f5ea6086d36d45d46db563493754334a3)

File: packages/server/beaver_triple_service/cmd/root.go (hunk 4 above)
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/cmd/root.go]: Corrected with [6e16de0](https://github.com/acompany-develop/QuickMPC/pull/123/commits/6e16de0e2447cd33e698136d8b92ee3ae3b61f4a)

File: packages/server/beaver_triple_service/go.mod (hunk 5 above)
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/go.mod]: Corrected with [e618c5c](https://github.com/acompany-develop/QuickMPC/pull/123/commits/e618c5ca260670501207788c42d95a52ba65bb6a)

File: packages/server/beaver_triple_service/cmd/generate_jwt.go (hunk 6 above)
Comment (KotaTakahashi9320) [reply packages/server/beaver_triple_service/cmd/generate_jwt.go]: Corrected with [bf2aa41](https://github.com/acompany-develop/QuickMPC/pull/123/commits/bf2aa416419195f2fc52f4f7c768dd4c4523ebeb)

**Review by arukuka — APPROVED** [review]
Thank you!

**Pull request merged**